### ISSUE TITLE
POI Selection in Unity Editor not working

### ISSUE THREAD

**Issue by Morbias86** [Microsoft/GalaxyExplorer#80]
Hi,
the selection of POIs and the Tools don't work in the newest version of Galaxy Explorer in Unity 5.5.0f3. Same goes for the Hololens Emulator. It works in the Hololens itself.
It works with the Commit a4f6c23 from Nov. 8th in Unity 5.5.0f3.
Greetings!

**Comment by elitemn** [comment]
Interesting, I'll take a look this weekend if someone else hasn't fixed it. I have a full schedule this week.


________________________________

**Comment by timGerken** [comment]
I'm out all next week and doubt I'll get to it tomorrow.

**Comment by elitemn** [comment]
Understood @timgerken. Feel free to assign this and or others to me.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by timGerken** [comment]
@ChimeraScorn , can you add @elitemn as someone to whom issues can be assigned for GE?

**Comment by elitemn** [comment]
I am working on this anyway.

**Comment by ChimeraScorn** [comment]
I’ve sent a request to elitemn adding him as a collaborator.  Once he accepts, we can assign the issue.

-C

**Comment by elitemn** [comment]
Accepted @ChimeraScorn

**Comment by elitemn** [comment]
@Morbias86  - 
HoloLens emulator (10.0.14393.0) works as expected with builds exported from Unity 5.5.0f3, as well as the HoloLens device. I can select POIs as normal, and I tested all of the tools (they work as expected). 
I can confirm that I am unable to select when running in the Unity editor.

Can you confirm your build of HoloLens emulator?

**Comment by Morbias86** [comment]
@elitemn -
I can confirm my build on HoloLens Emulator. Everything works as you described it, except for the selection in the Unity Editor. It's really weird.

**Comment by elitemn** [comment]
I think I found the issue.  standby.

**Comment by elitemn** [comment]
@Morbias86 - As a workaround while I investigate further: When playing in Unity player, In the Hierarchy, navigate to CoreSystems\Input.  In the inspector panel for "input", expand the "Gaze Selection" script and check the box for "Use Spherical Cone Search". This will override the code change that disabled selection in Unity.

**Comment by timGerken** [comment]
The code here fixed #59.
Wrapping the code inside the if with a #if !UNITY_EDITOR block fixes this issue and keeps #59 in-tact.
```
        if (!UnityEngine.VR.VRDevice.isPresent)
        {
#if !UNITY_EDITOR
            UseSphericalConeSearch = false;
#endif
        }
```

**Comment by timGerken** [comment]
resolved with #81
